java-manta-it/src/test/java/com/joyent/manta/client/MantaClientIT.java

```diff
@@ -374,7 +374,33 @@ public final void canMoveFile() throws IOException {
         Assert.assertEquals(movedContent, TEST_DATA);
     }
 
-    @Test
+    @Test(groups = "move")
+    public final void canMoveFileToDifferentUncreatedDirectory() throws IOException {
+        final String name = UUID.randomUUID().toString();
+        final String path = testPathPrefix + name;
+        mantaClient.put(path, TEST_DATA);
+        final String newDir = testPathPrefix + "subdir-" + UUID.randomUUID() + "/";
+
+        final String newPath = newDir + "this-is-a-new-name.txt";
+
+        mantaClient.move(path, newPath);
+        final String movedContent = mantaClient.getAsString(newPath);
+        Assert.assertEquals(movedContent, TEST_DATA);
+    }
+
+    @Test(groups = "move")
+    public final void canMoveFileToDifferentDirectory() throws IOException {
+        final String name = UUID.randomUUID().toString();
+        final String source = testPathPrefix + name;
+        mantaClient.put(source, TEST_DATA);
+
+        final String destination = testPathPrefix +
+                "stor/knoxmetrics-dev-marc/topics/+tmp/knox-metrics/partition=0/4ab7bccf-3f31-4b5c-bc74-35926084fd3d_tmp.parquet";
```

Comment:
The "knoxmetrics-dev-marc" might want to be scrubbed to something generic since it *sounds* like a product name.